### PR TITLE
feat: update antd to 4.24.12

### PR DESCRIPTION
**Pull request by love98ooo**

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@ant-design/icons": "4.6.2",
     "@craco/craco": "6.4.5",
-    "antd": "4.15.5",
+    "antd": "~4.24.0",
```
Comment (nomeguy) [inline web/package.json]: remove ~

File: web/src/App.js [hunk 2]
```diff
@@ -340,16 +338,21 @@ class App extends Component {
             // theme="dark"
             mode={"horizontal"}
             selectedKeys={[`${this.state.selectedMenuKey}`]}
-            style={{lineHeight: "64px"}}
+            style={{lineHeight: "64px", position: "absolute", left: 138, right: "300px"}}
```
Comment (nomeguy) [inline web/src/App.js:341]: What's this hack? Does Casdoor do this? Write better code. Copy Casdoor's solution

File: web/src/App.js [hunk 3]
```diff
@@ -340,16 +338,21 @@ class App extends Component {
...
           >
             {
               this.renderMenu()
             }
-            {
-              this.renderAccount()
-            }
             {/* <SelectLanguageBox /> */}
           </Menu>
+          {/* <Menu
```
Comment (nomeguy) [inline web/src/App.js]: Remove comments

**Review by nomeguy — CHANGES_REQUESTED** [review]

File: web/src/App.js (hunk 2 above)
Comment (love98ooo) [reply web/src/App.js:341]: image

Casdoor does this too.

**Review by nomeguy — APPROVED** [review]

**Pull request merged**